spec/services/alerts/importer_spec.rb

```diff
@@ -4,17 +4,15 @@
 
 RSpec.describe Alerts::Importer do
   subject(:importer) do
-    described_class.new(
-      profile: profile,
-      alerts: alerts
-    )
+    Moves::Importer.new(moves)
   end
 
-  let(:person) { create :person }
+  let(:person) { create :person, nomis_prison_number: 'G3239GV' }
```

Comment:
`nomis_prison_number` doesn't look like it needs to be explicit here. Could it be added to the factory so we have an `:prisoner` person type which automatically gets a faked NOMIS prison number?

Reply:
+1

Reply:
+1